auto_landing/CMakeLists.txt

```diff
@@ -7,20 +7,33 @@ catkin_simple(ALL_DEPS_REQUIRED)
 add_definitions(-std=c++17)
 set(CMAKE_BUILD_TYPE Release)
 
+find_package(Eigen3 REQUIRED)
```

Comment:
I don't think this is needed, as catkin_simple automatically reads and includes dependencies from package.xml.